Added attention for ALTER on Buffer table prior to 28.10.2020 (#27786)
ClickHouse/ClickHouse: docs/en/engines/table-engines/special/buffer.md

If you need to run ALTER for a subordinate table, and the Buffer table, we recommend first deleting the Buffer table, running ALTER for the subordinate table, then creating the Buffer table again.

!!! attention "Attention"

>     Running ALTER on Buffer table in releases made prior to 28 Sep 2020 will cause `Block structure mismatch` error (see [#15117](https://github.com/ClickHouse/ClickHouse/issues/15117)), so deleting Buffer table and then recreating it is the only option. Please check error is fixed in your release before trying to run ALTER on Buffer table.

Reviewer: ```suggestion
    Running ALTER on the Buffer table in releases made before 28 Sep 2020 will cause a `Block structure mismatch` error (see [#15117](https://github.com/ClickHouse/ClickHouse/issues/15117)), so deleting the Buffer table and then recreating is the only option. It is advisable to check that this error is fixed in your release before trying to run ALTER on the Buffer table.
```
